### ISSUE TITLE
[Phase 2 - Tools] Support for extension methods in VM service and observatory.

### ISSUE THREAD

**Issue by leafpetersen** [dart-lang/sdk#37889]
This tracks any required work for supporting debugging, profiling, stack traces, and related issues in the VM.

**Comment by leafpetersen** [comment]
@bkonyi is this on you, or can you point it at the right person?

**Comment by bkonyi** [comment]
I should be the right person for this. If it ends up being out of scope for me I'll delegate to the right person.

**Comment by a-siva** [comment]
CL for stack traces is out for review, once that lands the only remaining piece of work is support for extension method names in the break point command. e.g

break ClassName.MethodName works, similarly it should be possible to set breakpoints using
break ExtensionName.ExtensionMethodname.

**Comment by a-siva** [comment]
Support for demangled stack traces in stack traces and debugger has been added, there is a pending issue https://github.com/dart-lang/sdk/issues/38849 with respect to breakpoints which is a lower priority and will be addressed post D26
